website/package.json

```diff
@@ -100,7 +100,7 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-qunit": "^8.1.1",
     "fs-extra": "^11.1.1",
-    "jsdom": "^22.1.0",
+    "jsdom": "^25.0.1",
```

Comment:
I don't think we actually directly use this in the website any longer (removed [here](https://github.com/hashicorp/design-system/commit/e8cc0e7151642741e58b522431e0dc17429b27f9))

Reply:
Was able to remove it successfully without any issue